Switch/Checkbox/styles.scss

```diff
@@ -25,7 +25,7 @@
   line-height: ($grid * 4);
   padding-bottom: ($grid * 1);
   padding-left: ($grid * 10);
-  padding-top: ($grid * 1);
+  padding-top: ($grid * 1 - 1);
```

Comment:
1px = .2 grid. This can be expressed as `($grid * .8)`. I can push a patch it you want (the same applies to line 47)